test/nodes/test_llm_streaming.py

```
@@ -0,0 +1,621 @@
+#
+# ``streaming_config.py`` has zero engine dependencies.
+# ``streaming.py`` uses a relative import from ``streaming_config`` and a
+#   lazy ``from ai.common.schema import Answer`` inside ``stream_response``.
+#   The lazy import is patched in tests that exercise that path.
+# ---------------------------------------------------------------------------
+
+_REPO = Path(__file__).resolve().parent.parent.parent
+_LLM_BASE = _REPO / 'nodes' / 'src' / 'nodes' / 'llm_base'
+
+
+def _load_module(name: str, path: Path):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_load_module`

(ANN202)